src/esssans/isis/io.py

```diff
@@ -31,10 +31,6 @@ class Filename(sciline.Scope[RunType, str], str):
 """1-D variable listing all masked detector IDs."""
 
 
-def to_path(filename: FilenameType, path: DataFolder) -> FilePath[FilenameType]:
-    return f'{path}/{filename}'
-
-
```

Comment:
Why was this removed? This is needed for generic instrument-independent (and pooch-independent) setup. My thought was that this would actually move to the top level eventually, i.e., not remain in the `isis` module.

Reply:
I put this back in when I made a common `data.py` file. See if that works.